setup.py

```diff
@@ -201,6 +198,7 @@ def run(self):
 
 if is_manylinux:
     data = ['capi/libonnxruntime_pywrapper.so'] if nightly_build else []
+    data += [path.join('capi', x) for x in dl_libs if path.isfile(path.join('onnxruntime', 'capi', x))]
```

Comment:
quick question. i didn't see that capi/libonnxruntime_pywrapper.so in the ort-gpu-nightly package I was testing. is that expected?

Reply:
Yes. It is for PyOP. We didn't enable it.